views/docs/console.md

```diff
@@ -52,6 +52,22 @@ Note that if you are using Asterisk, the format is slightly different. The examp
 AHN> Adhearsion::OutboundCall.originate 'SIP/user1'
 </pre>
 
+As well, if you need to place calls out into the outside world, you may get an error message 
+such as "I don't know how to authenticate..." if you try to call directly. 
```

Comment:
Ben,

Here is what I did from the AHN console
Adhearsion::OutboundCall.originate 'IAX2/4443@toronto.voip.ms'

And Asterisk gave back:
chan_iax2.c:11243 socket_process_helper: I don't know how to authenticate
4443 to 184.75.215.106

Johannes

On Wed, Nov 11, 2015 at 10:28 AM, Ben Klang <notifications@github.com>
wrote:

> In views/docs/console.md
> <https://github.com/adhearsion/adhearsion-website/pull/37#discussion_r44544376>
> :
>
> > @@ -52,6 +52,22 @@ Note that if you are using Asterisk, the format is slightly different. The examp
> >  AHN> Adhearsion::OutboundCall.originate 'SIP/user1'
> >  </pre>
> >
> > +As well, if you need to place calls out into the outside world, you may get an error message
> > +such as "I don't know how to authenticate..." if you try to call directly.
>
> Would it be possible to include an example of the exact error message? I
> admit I'm not familiar with this one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adhearsion/adhearsion-website/pull/37/files#r44544376>
> .
>